tests/autoyast/clone.pm

```diff
@@ -19,11 +19,18 @@
 use strict;
 use base 'console_yasttest';
 use testapi;
+use version_utils 'is_sle';
```

Comment:
I think this is unused now. Please crosscheck and remove

Reply:
Ok, I will remove this line.